### DISCUSSION TITLE
2022/06/19/firefox-custom-ca/

### DISCUSSION THREAD

**Discussion by giscus** [jdeluyck/kcoreorg-website#141] [Blog Comments]
# 2022/06/19/firefox-custom-ca/

In the past (before the introduction of the new design/engine/…), you were able to import Certificate Authority certificates without problems. The new version took that away, unfortunately.

https://kcore.org/2022/06/19/firefox-custom-ca/

**Comment by 3v1n0** [comment]
Thanks for sharing, indeed there's such setting.

It doesn't seem to pick my certificate, but at least there's some room...
